service/src/test/kotlin/no/nav/su/se/bakover/service/behandling/UnderkjennBehandlingTest.kt

```diff
@@ -473,4 +485,185 @@ class UnderkjennBehandlingTest {
+                        ektemakeEllerSamboerUførFlyktning = false,
+                        begrunnelse = null
+                    )
+                )
+            )
+
+        val simulering = Utbetaling.SimulertUtbetaling(
+            id = UUID30.randomUUID(),
+            opprettet = Tidspunkt.now(),
+            sakId = sakId,
+            saksnummer = saksnummer,
+            fnr = fnr,
+            utbetalingslinjer = emptyList(),
+            type = Utbetaling.UtbetalingsType.NY,
+            behandler = saksbehandler,
+            avstemmingsnøkkel = Avstemmingsnøkkel(Tidspunkt.now()),
+            simulering = Simulering(
+                gjelderId = fnr,
+                gjelderNavn = "lol",
+                datoBeregnet = 1.januar(2021),
+                nettoBeløp = 0,
+                periodeList = listOf()
+            ),
+        )
+
+        val behandlingRepoMock = mock<BehandlingRepo> {
+            on { hentBehandling(any()) } doReturn behandling
+        }
+
+        val utbetalingServiceMock = mock<UtbetalingService> {
+            on { simulerUtbetaling(any(), any(), any()) } doReturn simulering.right()
+        }
+
+        val actual = BehandlingTestUtils.createService(
+            behandlingRepo = behandlingRepoMock,
+            utbetalingService = utbetalingServiceMock,
+            microsoftGraphApiOppslag = BehandlingTestUtils.microsoftGraphMock.oppslagMock
+        ).simuler(
+            behandlingId = behandling.id,
+            saksbehandler = saksbehandler
+        )
+
+        actual shouldBe behandling.copy(
+            status = Behandling.BehandlingsStatus.SIMULERT,
+        ).right()
+    }
+
+    @Test
+    fun `kan sende til attestering med status UNDERKJENT`() {
+        val behandling: Behandling =
+            innvilgetBehandlingTilAttestering.copy(
+                status = Behandling.BehandlingsStatus.UNDERKJENT
+            )
+
+        val behandlingRepoMock = mock<BehandlingRepo> {
+            on { hentBehandling(any()) } doReturn behandling
+        }
+        val personServiceMock: PersonService = mock {
+            on { hentAktørId(any()) } doReturn aktørId.right()
+        }
+        val oppgaveServiceMock = mock<OppgaveService> {
+            on { opprettOppgave(any()) } doReturn nyOppgaveId.right()
+            on { lukkOppgave(any()) } doReturn Unit.right()
+        }
+        val behandlingMetricsMock = mock<BehandlingMetrics>()
+
+        val actual = BehandlingTestUtils.createService(
+            behandlingRepo = behandlingRepoMock,
+            personService = personServiceMock,
+            oppgaveService = oppgaveServiceMock,
+            behandlingMetrics = behandlingMetricsMock,
+            microsoftGraphApiOppslag = BehandlingTestUtils.microsoftGraphMock.oppslagMock
+        ).sendTilAttestering(behandlingId = behandling.id, saksbehandler = saksbehandler)
+
+        actual shouldBe behandling.copy(
+            status = Behandling.BehandlingsStatus.TIL_ATTESTERING_AVSLAG,
```

Comment:
Denne synes jeg ser litt spennende ut.. hvordan ender vi opp med at en "innvilgetBehandlingTilAttestering" (`status = TIL_ATTESTERING_INNVILGET`) som har blitt sendt tilbake fra attestering (`status = UNDERKJENT`) ender opp som `TIL_ATTESTERING_AVSLAG`?

Se min tidligere kommentar om `erInnvilget()` for årsaken til dette.

Reply:
Riktig. forvirret meg selv når jeg sjekket noen ting med hvordan behandlingen var satt opp.